content/body/form-error-checking.php

```diff
@@ -23,12 +22,9 @@
       ]); ?>
 
       <p>
-        You can use just <code>required</code> and
-        <code>pattern</code> attributes on HTML forms to do client side
-        validation <strong>without JavaScript</strong>.  However, in order to make the messaging
-        more accessible, we have added a tiny bit of JavaScript code (inspired by <a href="https://pauljadam.com/guides/html5-form.html">this accessible HTML5 forms code demo by Paul J Adam</a>) in order to ensure the error messages
-        themselves are more accessible to screen reader users (see the last step in the code
-        walkthrough for details)
+        You can use just the <code>required pattern </code> attributes on HTML forms to do client-side validation <strong>without JavaScript</strong>. 
```

Comment:
The `<code>` block should only be around the "required" text, with a separate `<code>` block around the "pattern" text, since those are the exact attribute names.

```suggestion
        You can use just the <code>required</code> and <code>pattern</code> attributes on HTML forms to do client-side validation <strong>without JavaScript</strong>. 
```